<ui version="4.0" >
 <class>XmlServerClass</class>
 <widget class="QWidget" name="XmlServerClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>XmlServer</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>ConnectTimeOut</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>30</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>(hour)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lTime" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>30</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>210</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount" >
    <number>4</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="SaveWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>70</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount" >
    <number>2</number>
   </property>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="SaveB" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateB" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>150</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CutB" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ENDB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>END</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
